src/main/java/com/example/spring/dto/book/BookDto.java

```diff
@@ -1,9 +1,14 @@
 package com.example.spring.dto.book;
 
 import java.math.BigDecimal;
-import lombok.Data;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
 
-@Data
+@Getter
+@Setter
+@ToString
 public class BookDto {
     private Long id;

```

Comment:
what about validations?

Reply:
No need to validate, because only valid data can get to bookdto